model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -103,7 +101,7 @@ def __init__(
         self._grid = grid
         self._decomposition_info = decomposition_info
         self._attrs = metadata
-        self._geometry_type: base.GeometryType = grid.global_properties.geometry_type
+        self._geometry_type: base.GeometryType = base.GeometryType.ICOSAHEDRON
```

Comment:
Is this a necessary change?

Reply:
Yes, good point and fully agree.

I ask because this will clash with #853 which requires actually having this correctly set. I just wanted to understand the motivation, so let's go with this now and I'll get this cleaned up in #853.